### ISSUE TITLE
Add option to change calendar/list of existing tasks in properties sidebar

### ISSUE THREAD

**Issue by noresistence** [nextcloud/tasks#97]
When I create a task, it will often be created in the wrong calendar. (1)

I then need a way to set the correct calendar for the task. As there is no visible option for doing this in the UI (2), I propose to __add this option to the task properties menu__.

(1): That is because it is cumbersome and slow to always select the correct calendar on the left menu before adding a new task in the main window. Slow due to the reload of the filtered task list; even with good internet connection it is an annoying delay. Furthermore, i do not like nor need the filtered task list when adding a new task, as i usually go on with adding a task for another calendar.

(2) I discovered just today, after months, that it is possible to use drag and drop. This does not solve my issue because
* it is not intuitive (at least, has not been for me) - there is no indication that the tasks may be reordered
* like for the calandar, there should also be a way to set this as part of the right-hand properties menu.
* in a long list of tasks, distributed over more than 4 calendars, drag&drop does not perform very well.

**Comment by raimund-schluessler** [comment]
@jancborchardt What do you think? Is this really necessary?

**Comment by jancborchardt** [comment]
I agree that it would be good to have a method other than drag & drop to move tasks to another list. For accessibility but also for discoverability reasons. The solution of having it in the right sidebar seems good – below the task title, similar to the Calendar app.

**Comment by hubutz** [comment]
Hi,
I have the same issue. I need a way to move tasks out of my "Inbox" list into the list they belong too. And I do not find a way to do so. The drag'n'drop way seems not to work (or i might be stupid).

Any idea when this will/might happen?

**Comment by raimund-schluessler** [comment]
As of da26201cf6d4afa139ba5fdc6782059de5002262 this is implemented in the vue rewrite branch.
